coding/atoms.dm

```diff
@@ -127,4 +133,8 @@ atom
 				if(!forced)
 					if(loc) loc.Entered(src, oldloc)
 					if(area2 && area1 != area2 && !isarea(loc))
-						area2.Entered(src, oldloc)
\ No newline at end of file
+						area2.Entered(src, oldloc)
+
+		Bump(atom/A)
+			A.Bumped(src)
```

Comment:
Now when a movable atom bumps any other atom, calls that atom's Bumped proc. As far as I could tell, this proc was never called anywhere in the code, despite having been implemented in several different objects. Not sure if this is a recent bug or something, real head scratcher.